src/main/java/bammerbom/ultimatecore/bukkit/commands/CmdJail.java

```diff
@@ -116,7 +116,11 @@ public void run(final CommandSender cs, String label, String[] args) {
             Location loc = UC.getServer().getJail(jail);
             if (pl.isOnline()) {
                 LocationUtil.teleportUnsafe(pl.getPlayer(), loc, TeleportCause.PLUGIN);
-                r.sendMes(pl.getPlayer(), "jailTarget", "%Time", time != -1L ? DateUtil.format(time + System.currentTimeMillis()) : r.mes("jailEver"));
```

Comment:
I had to change this line to a better interpretation of the message in Spanish. I hope it's not a problem